### PR TITLE
#152 Unit tests for Account controller

### PR DESCRIPTION
**Pull request by swem-st**

### CODE REVIEW

File: UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+using Moq;
+using Xunit;
+
+namespace UnitTestApp.Tests.Unit.WebApi.Controllers
+{
+    public class AccountControllerUnitTest
+    {
+        private readonly Mock<IAccountService> _accountService = new Mock<IAccountService>();
+        private readonly AccountController accountController;
+        private readonly ChangePasswordRequestApiModel changePasswordRequestApiModel;
```
Comment (pinkevmladchy) [inline UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs:22]: add empty line

File: UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+            };
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+            {
+                new Claim("Id", "anuviswan"),
+
+            }, "mock"));
+
+            accountController.ControllerContext.HttpContext = new DefaultHttpContext() { User = user };
+        }
+
```
Comment (pinkevmladchy) [inline UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs:40]: delete empty line

File: UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+
+        [Fact]
+        public async Task UpdatePersonalData_ReturnsSuccess()
+        {
+            //Arrange
+
```
Comment (pinkevmladchy) [inline UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs]: delete empty line

File: UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+            var result = await Assert.ThrowsAsync<RestException>(act);
+            Assert.Equal(HttpStatusCode.BadRequest, result.StatusCode);
+            Assert.Equal("Failed", result.Message);
+        }
+
+        [Fact]
+        public async Task GetPersonalData_ReturnsSuccess()
+        {
+            //Arrange
+
```
Comment (pinkevmladchy) [inline UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs]: delete empty line

File: UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+            var result = await Assert.ThrowsAsync<RestException>(act);
+            Assert.Equal(HttpStatusCode.NotFound, result.StatusCode);
+            Assert.Equal("Failed", result.Message);
+        }
+
+        [Fact]
+        public async Task GetUserById_ReturnsSuccess()
+        {
+            //Arrange
+
```
Comment (pinkevmladchy) [inline UnitTestApp.Tests/Unit/WebApi/Controllers/AccountControllerUnitTest.cs]: delete empty line

**Review by pinkevmladchy — APPROVED** [review]

**Review by SashaKorniichuk — APPROVED** [review]

**Pull request merged**